src/main/java/org/powerflows/dmn/io/yaml/CustomRepresenter.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2018-present PowerFlows.org - all rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.powerflows.dmn.io.yaml;
+
+import org.powerflows.dmn.engine.model.decision.expression.ExpressionType;
+import org.powerflows.dmn.io.yaml.model.YamlDecision;
+import org.powerflows.dmn.io.yaml.model.rule.YamlRuleEntry;
+import org.yaml.snakeyaml.introspector.Property;
+import org.yaml.snakeyaml.nodes.Node;
+import org.yaml.snakeyaml.nodes.NodeTuple;
+import org.yaml.snakeyaml.nodes.Tag;
+import org.yaml.snakeyaml.representer.Represent;
+import org.yaml.snakeyaml.representer.Representer;
+
+import java.util.Collection;
+import java.util.Map;
+
+public class CustomRepresenter extends Representer {
+    CustomRepresenter() {
+        this.setPropertyUtils(new CustomPropertyUtils());
+        this.addClassTag(YamlDecision.class, Tag.MAP);
+        this.representers.put(YamlRuleEntry.class, new RepresentYamlRuleEntry());
+    }
+
+    class RepresentYamlRuleEntry implements Represent {
```

Comment:
Should be @Override and final argument

Reply:
@override is for methods not classes :P

Reply:
True, and here you override public Node representData(Object data)

Reply:
then why comment on class not method?

Reply:
fixed